src/composables/multisigAccounts.ts

```diff
@@ -62,45 +62,68 @@ export function useMultisigAccounts({ store }: UseMultisigAccountsOptions) {
 
   const multisigProgress = ref<IMultisigCreationStep>(MULTISIG_CREATION_STEPS.preparing);
 
+  async function createMultisigContractInstance() {
+    const drySdk = await getDrySdk();
+    return drySdk.getContractInstance({
+      aci: SimpleGAMultiSigAci,
+      bytecode: SimpleGAMultiSigBytecode,
+    });
+  }
+
+  /**
+   * Estimate the amount of AE tokens to be payed for the multisig account creation.
+   */
+  async function estimateMultisigAccountDeployGasFee(
+    multisigContractInstance: any,
+    noOfConfirmations: number,
+    signersAddresses: string[],
+  ): Promise<number> {
+    return (multisigContractInstance || await createMultisigContractInstance())
+      ._estimateGas('init', [noOfConfirmations, signersAddresses]);
+  }
+
   /**
    * @param {number} noOfConfirmations no of minimum confirmation needed to process a transaction
    * @param {string[]} signers
    */
   async function deployMultisigAccount(
     noOfConfirmations: number,
-    signersAddresses: [string, string, ...string[]],
+    signersAddresses: string[],
```

Comment:
Yes, but this requirement we need to test in the logic, not on the code validation level. So we should be able to add only one signer, but when creating the vault we need to validate this. This is why we have the `MULTISIG_VAULT_MIN_NUM_OF_SIGNERS` constant.

Reply:
I agree with that but since this code in the composables and it mostly acts like a library. In the future this function may be reused by other developers hence I guess this type definition is also important along with the logic condition too.